### PR TITLE
feat(users-status): separate user enabled from user status and toggle the user status in settings page

### PR DESCRIPTION
**Pull request by khodedawsh**
enabling and disabling a user be handled by new api methods rather than doing it using modify_user method

### UI Specification

- [X] User enable and disable button in user settings
- [X] English translation 
- [x] Soranî translation

### CODE REVIEW

**Review by Krr0ptioN — APPROVED** [review]
LGTM

File: dashboard/src/features/users/services/user-status.mutate.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+} from "@marzneshin/features/users";
+
+
+interface UsersStatusEnabledQuery {
+    user: UserMutationType;
+    enabled: boolean;
+}
+
+export async function userStatusEnabled({ user, enabled }: UsersStatusEnabledQuery): Promise<UserMutationType> {
+    const action = enabled ? 'enable' : 'disabled'
```
Comment (khodedawsh) [inline dashboard/src/features/users/services/user-status.mutate.ts:16]: It should be `disable`, without a d.

**Comment by khodedawsh** [comment]
after clicking on enable/disable button the user object should be updated so that the button action switches (disable->enable) as soon as server sends the response

**Review by Krr0ptioN — APPROVED** [review]
Accepted

**Pull request merged**